registry/Dockerfile

```
@@ -18,6 +18,9 @@ RUN GO111MODULE=off CGO_ENABLED=0 make clean binaries VERSION=${REGISTRY_VERSION
 # Stage2: setup runtime container
 FROM quay.io/cybozu/ubuntu:20.04
 
+RUN apt-get update && apt-get install -y \
+    jq
```

You can shrink the container image by removing package list files created by `apt-get update`.
In my own test, the image was downsized from 172 MB to 135 MB.
cf. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get
cf. https://github.com/cybozu/neco-containers/blob/main/bird/Dockerfile and other Dockerfiles

Note:
* `apt-get clean` is automatically executed
* `--no-install-recommends` is also useful in general; it makes no difference for `jq`, though
* The cleanup is meaningful only for the final stage; it does not have meaning for intermediate containers

```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends \
    jq && \
    rm -rf /var/lib/apt/lists/*
```

---

Thank you for the review.
Reduce container image at 68309c324b29fa94532b777c6b9d33de785f9e06.